dough-api/pom.xml

```diff
@@ -269,9 +273,9 @@
 
         <!-- PreciousStones -->
         <dependency>
-            <groupId>com.github.marcelo-mason</groupId>
+            <groupId>net.sacredlabyrinth.Phaed</groupId>
```

Comment:
Would rather not rely on another 3rd party repo. Jitpack is reliable. So why change this?

Reply:
Sorry I didn't realize it was in jitpack, when I tried to build it said this was missing so I put in the repository listed in the [repository github](https://github.com/marcelo-mason/PreciousStones/wiki/Developers-API)